docs/advanced-guide/custom-spans-in-tracing/page.md

```diff
@@ -31,5 +31,4 @@ func MyHandler(c context.Context) error {
 In this example, **my-custom-span** is the name of the custom span that is added to the request.
 The defer statement ensures that the span is closed even if an error occurs to ensure that the trace is properly recorded.
 
-
-
+> ##### Check out the example of creating a custom span in GoFr : [Visit Github](https://github.com/gofr-dev/gofr/blob/0730b9496b942b318d51139a561f3a5f90c7dbbf/examples/http-server/main.go#L58)
```

Comment:
- space should not be present before `:`
- we should use links from `main` branch
- also, we can use the similar statement to refer example, currently almost every example has different lines (which feels inconsistent)

Reply:
I added redirection to directly traceHandler function instead of whole example as the file also had normal handler, redis, mysql and error handler examples.

Reply:
yes, that is correct! But, the link should be a branch (preferably `main`) if not main then maybe `development`, instead of a commit hash.
Like: `https://github.com/gofr-dev/gofr/blob/development/examples/http-server/main.go#L58`